modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -2816,6 +2818,8 @@ definition {
 			urlAction = "http://localhost:8080",
 			whenAction = "On After Add");
 
+		Button.clickSave();
+
```

Comment:
Hey @victorhcunha, 

it is more assertive and cleaner to use  the **ObjectAdmin.addObjectActionViaUI**

The failure happens because there is a **WaitForElementNotPresent** on **addObjectActionViaUI** after the save, but the objective here is the opposite of that.

Refactor the macro using: **WaitForElementPresent(locator1 = "ObjectAdmin#SIDE_PANEL");**